### ISSUE TITLE
[Bug] Accessibility features filtering does not work

### ISSUE THREAD

**Issue by plunkettgoogle** [CityOfDetroit/bloom#901]
STEPS TO REPRODUCE:
1. Deploy locally, go to http://localhost:3000/listings/filtered?page=1&elevator=true

EXPECTED RESULTS:
All listings with the `elevator` feature should show.

OBSERVED RESULTS:
No listings are shown, and a raw polyglot string key appears:
![localhost_3000_listings_filtered_page=1 elevator=true](https://user-images.githubusercontent.com/66751489/146986004-e014b4ad-5502-4ced-86ab-436668862d06.png)

ADDITIONAL INFORMATION:
The raw polyglot string key appears because a [`listingsError`](https://github.com/CityOfDetroit/bloom/blob/main/sites/public/pages/listings/filtered.tsx#L56) is returned by the backend. We don't currently handle this case, which is a separate issue.
